### PR TITLE
Update dg v1.1

### LINKED ISSUE #17: Update the DG use cases section

### PR DESCRIPTION
**Pull request by Jayne1010**
Closes #17

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -275,26 +275,28 @@ _{Explain here how the data archiving feature will be implemented}_
 **Target user profile**:
 
 * has a need to manage a significant number of contacts
+* GPs/receptionist at family clinics to organise patients contacts quickly.
```
Comment (tengcharmaine) [inline docs/DeveloperGuide.md:278]: Is this line supposed to be here?

**Review by tengcharmaine — COMMENTED** [review]
Just 1 comment else it LGTM.

**Comment by drustanyjt** [comment]
@Jayne1010  I think this has the same issue as #23 with the commit subject not being imperative.
In particular the commit should be start with something like `Upgrade` instead of `Upgraded`
image

Is it possible to edit the commit message in Git?

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Jayne1010) [reply docs/DeveloperGuide.md:278]: yes

**Pull request closed without merging**